UI/SidePanel.cpp

```diff
@@ -1136,12 +1145,25 @@ void SidePanel::PlanetPanel::RefreshPlanetGraphic() {
         m_planet_graphic->Play();
     } else if (planet->Type() < NUM_PLANET_TYPES) {
         int planet_image_sz = PlanetDiameter(planet->Size());
+        planet_graphic_height = GG::Y(MaxPlanetDiameter());
         GG::Pt planet_image_pos(GG::X(MaxPlanetDiameter() / 2 - planet_image_sz / 2 + 3),
                                 GG::Y(MaxPlanetDiameter() / 2 - planet_image_sz / 2));
         m_rotating_planet_graphic = new RotatingPlanetControl(planet_image_pos.x, planet_image_pos.y,
                                                               m_planet_id, m_star_type);
         AttachChild(m_rotating_planet_graphic);
     }
+
+    // add blockade graphic
+    if (!GetSupplyManager().SystemHasFleetSupply(planet->SystemID(), planet->Owner()) && (!planet->Unowned())) {
```

Comment:
Isn't a planet always a source for fleet supply as long as it isn't blockaded? If not, how else would I determine if a planet is blockaded?